README.md

```diff
@@ -1,8 +1,36 @@
 ## Orange DAM PHP API client
 
-The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to safely and securely extend the capabilities of their Orange DAM platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with other software.
+The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to extend the capabilities of their Orange digital asset management platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with a CMS or other software/workflows.
```

Comment:
We are using DAM specific portions of the API, but this library could be for products beyond just the DAM.

That calls into question the very name of this library though... 🤔 

Something I don't think we fully realized when we started this. Let me ask OL tomorrow when we chat for 100% clarity on this though.

Reply:
Yeah, I think we leave it for now.